setup.py

```diff
@@ -7,7 +7,9 @@
     install_requires = [
         "octopus==1.0.0",
         "esprit",
-        "Flask"
+        "Flask",
+        "setproctitle",
```

Comment:
So when adding a dependency I will either install it, and then add `==theversion` to requirements.txt or setup.py, or I will look it up on pypi first to see the latest version and freeze to that.